scripts/common.py

```diff
@@ -58,18 +59,19 @@
 # there is a 'make test' bug in redis 6.0.6 where the solution has not been
 # released. This is explained in this issue:
 # https://github.com/redis/redis/issues/7540.
-# IMPORTANT STEPS TO DEVELOPERS FOR UPGRADING REDIS:
+# IMPORTANT STEPS FOR DEVELOPERS TO UPGRADE REDIS:
 # 1. Download the new version of the redis cli.
-# 2. Extract it in the folder it was downloaded, most likely Downloads/.
+# 2. Extract the cli in the folder that it was downloaded, most likely
+#    Downloads/.
 # 3. Change directories into the folder you extracted, titled
 #    redis-<new version>/ and change into that directory:
 #    cd redis-<new version>/
 # 4. From the top level of the redis-<new version> directory,
 #    run 'make test'.
 # 5. All of the tests should pass with an [ok] status with no error codes. The
 #    final output should be 'All tests pass.'
-# 6. Make sure all of the tests pass before committing the upgrade to develop.
-#    Be sure to indicate in your PR that all of the 'make test' tests pass.
+# 6. Be sure to leave a note in the PR description to confirm that you have read
+#    this message, and that all of the make test tests pass.
```

Comment:
Put quotes around "make test", otherwise the sentence does not make sense.

Reply:
Ah I see, you just copied my comment. Note that I wrote `make test` with backticks. Please make sure to read the final thing to ensure it's fully clear, don't just copy reviewers' comments blindly (which may have been written in a hurry anyway). Ultimately you should be responsible for making sure that what you write is clear to whoever is reading it.

Reply:
Ah I see. Fixed.